arenadata/Dockerfile

```diff
@@ -1,5 +1,10 @@
 FROM rockylinux:8.8 as base
```

Comment:
how about newadded ubuntu dockerfile?

Reply:
This is not part of a ticket and should be done in a separate task

Reply:
I think it's pretty easy to do in this same patch.